### ISSUE TITLE
Add support for Claude by Anthropic

### ISSUE THREAD

**Issue by malfoyslastname** [luminai-companion/agn-ai#107]
https://www.anthropic.com/product

From what I understand, Claude is:
- Temporarily free
- On-request access, but supposedly very easy to get into
- Very high quality

This makes it a very promising adapter to add to agnai.
I've just requested access, myself.

Once I have access, I'm willing to make a PR if you give the OK.

**Comment by malfoyslastname** [comment]
Looked into poe to write an adapter. I was banking on being able to enter the entire defs and conversation history in a single message, but found out messages silently fail if they are over 8000 characters (~1600 tokens).

That means:
(1) no editing messages
(2) gaslight and character defs will eventually leave the context window
(3) no "ultimate jailbreak" techniques

Under those conditions I'm not convinced writing an adapter is worth the effort. Better to just use the poe interface.

In conclusion:
- Claude API: paid and difficult to get in
- Poe: an adapter would be difficult to write and not offer much value

Closing, feel free to re-open though.

**Comment by malfoyslastname** [comment]
I just realized that since the Poe interface allows you to delete individual messages from the middle of the conversation, one could estimate when the character defs are about to leave the context window and programmatically remove them from the conversation. Could mitigate problem (2), but not sure that this kind of hack is worth managing on the agnai side.

**Comment by malfoyslastname** [comment]
Update: Anthropic started giving API access, many people have gotten it. Seems free for the time being. I have access. I'm starting work on an adapter today.
